manifests/charts/istio-control/istio-discovery/files/waypoint.yaml

```diff
@@ -12,6 +12,14 @@ metadata:
         "gateway.networking.k8s.io/gateway-name" .Name
         "istio.io/gateway-name" .Name
       ) | nindent 4 }}
+  {{- if ge .KubeVersion 128 }}
+  # Safe since 1.28: https://github.com/kubernetes/kubernetes/pull/117412
```

Comment:
btw, did you verify this? I never really understood how that PR fixed the problem...

Reply:
Yes I did. A pod can be deleted immediately if mounting is failed